src/foundry/foundry.js/pixi/containers/placeableObjects/token.d.ts

```diff
@@ -98,21 +44,23 @@ declare class Token extends PlaceableObject<Token.Data> {
   static get embeddedName(): 'Token';
 
   /**
-   * Apply initial sanitizations to the provided input data to ensure that a Token has valid required attributes.
+   * Establish an initial velocity of the token based on it's direction of facing.
+   * Assume the Token made some prior movement towards the direction that it is currently facing.
    */
-  protected _cleanData(): void;
+  protected _getInitialVelocity(): { dx: number; sx: number; dy: number; sy: number };
 
   /**
-   * @remarks
-   * Not implemented by Token
+   * A convenient reference to the Actor object associated with the Token embedded document.
    */
-  get bounds(): never;
+  get actor(): Actor | null;
```

Comment:
I guess this should be changed?

Reply:
Yeah I think that should be the configured Actor class.

Reply:
I redefined it like this:
```typescript
get actor(): ConfiguredDocumentClass<typeof Actor> | null;
```

When I add this test I get an erorr:
```typescript
expectType<Actor | null>(token.actor);
// Argument of type typeof Actor | null is not assignable to parameter of type Actor | null.
// Type typeof Actor is missing the following properties from type Actor: img, parent, pack, data, and 62 more.
```

So i guess I need to change it to 
```typescript
get actor(): ConfiguredDocumentClass<Actor> | null;
```
but then I get an error:
```
Type 'Actor' does not satisfy the constraint 'DocumentConstructor'.
  Type 'Actor' is missing the following properties from type 'Pick<typeof Document, "metadata" | "schema" | "create" | "prototype" | "database" | "implementation" | "collectionName" | "documentName" | "canUserCreate" | "createDocuments" | "updateDocuments" | "deleteDocuments">': metadata, schema, create, prototype, and 6 more.
```

What did I do wrong?

Reply:
Never mind, got it :-)